torch/_dynamo/trace_rules.py

```diff
@@ -35,40 +40,2732 @@
 and trace_rules.lookup consolidation is done. Then the explicit listing of skip/inline objects have
 a higher priority, which can be used to override the skipfiles.check rules in some cases.
 """
-torch_name_rule_map = {
+manual_torch_name_rule_map = {
+    "torch.default_generator#get_state": TorchInGraphFunctionVariable,
```

Comment:
What does the # mean here?

Reply:
```get_state``` is a method rather than a function, so we need different identifier telling ```load_object``` to properly load it. This is rare cases for in graph functions.